docs/docs/reference/metaprogramming/macros.md

```diff
@@ -62,14 +61,12 @@ Quotes and splices can also be applied directly to identifiers. An identifier
 splice `${x}`. Analogously, an quoted identifier `'x` that appears inside a splice
 is treated as a quote `'{x}`. See the Syntax section below for details.
 
-Quotes and splices are duals of each other. For arbitrary
-expressions `e` and types `T` we have:
+Quotes and splices are duals of each other.
+For arbitrary expressions `e` we have:
 
 ```scala
 ${'{e}} = e
 '{${e}} = e
```

Comment:
Is this now actually true? At some point, some normalization was observable .

Reply:
That was in the patterns, not here.